now.json

```
@@ -1,63 +1,27 @@
 {
+  "version": 2,
   "alias": "nextjsorg.now.sh",
-  "type": "static",
-  "static": {
-    "directoryListing": false,
-    "headers": [
-      {
-        "source": "/_next/static/chunks/*",
-        "headers": [
-          {
-            "key": "Cache-Control",
-            "value": "public, max-age=31536000, immutable"
-          }
-        ]
-      },
-      {
-        "source": "/_next/static/runtime/*",
-        "headers": [
-          {
-            "key": "Cache-Control",
-            "value": "public, max-age=31536000, immutable"
-          }
-        ]
-      },
-      {
-        "source": "/_next/static/commons/**",
-        "headers": [
-          {
-            "key": "Cache-Control",
-            "value": "public, max-age=31536000, immutable"
-          }
-        ]
-      },
-      {
-        "source": "/static/favicon/**",
-        "headers": [
-          {
-            "key": "Cache-Control",
-            "value": "public, max-age=31536000, immutable"
-          }
-        ]
-      },
-      {
-        "source": "/static/images/**",
-        "headers": [
-          {
-            "key": "Cache-Control",
-            "value": "public, max-age=31536000, immutable"
-          }
-        ]
-      },
-      {
-        "source": "/_next/static/*/pages/**/*.js",
-        "headers": [
-          {
-            "key": "Cache-Control",
-            "value": "public, max-age=31536000, immutable"
-          }
-        ]
+  "builds": [
+    { "src": "package.json", "use": "@now/static-build" }
+  ],
+  "routes": [
+    {
+      "src": "/_next/static/(chunks|runtime|commons)/*",
```

Since this is regex, it has to be this:

```suggestion
      "src": "/_next/static/(chunks|runtime|commons)/.*",
```

---

Same goes for the other route matchers